drizzlepac/haputils/get_git_rev_info.py

```diff
@@ -108,10 +107,10 @@ def get_rev_id(local_repo_path):
             else:
                 raise ValueError("Git revision information not found.")
     except Exception:
-        pass
+        log.warning("Problem encountered getting git revision ID")
+        sys.exit(1)
```

Comment:
Since this Exception has been promoted to be more serious, I have to ask if this failure serious enough to stop all processing and exit?  Shall we (Warren, Mike, Michele) discuss this?

Reply:
So the interesting thing here is that the git revision id isn't reported by this piece of code until after the SVM code as basically finished processing, and not even directly by either pipeline. it's actually invoked during by the SVM QA test that compares HAP-created sourcelists to their HLA classic counterparts, so it's a very very very specific failure mode that won't occur for observations that weren't processed by HLA, or if a certain environment variable exists and is set to "True". So for all these reasons, you are proably right, @mdlpstsci that the severity on the logging statement in question could be knocked down a notch from "error" to "warning".
All of this sort of begs the question though...is the git revision ID reported earlier in the code in a place that it will always be logged? this seems like a good thing to have in either the log files or somewhere in the headers of the output products.

Reply:
> @dulude I agree. The Git tag is put in the output log of the HSTCAL processing (calxxx) pipelines. I do think this is extremely useful to do for the HAP processing, but this is an issue for discussion and a different PR!

Reply:
@dulude I agree. The Git tag is put in the output log of the HSTCAL processing (calxxx) pipelines. I do think this is extremely useful to do for the HAP processing, but this is an issue for discussion and a different PR!

Reply:
ok. I'll pull out the 'sys.exit()' and decrease the severity of the logging statement from "error" to "warning".

Reply:
I created Ticket [HLA-718](https://jira.stsci.edu/browse/HLA-718) to deal with this problem as per your suggestion.

Reply:
Issue resolved by commit [9a25d71](https://github.com/spacetelescope/drizzlepac/pull/1267/commits/9a25d7117646588b44d958d25db2572acdf0887d)